uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -510,3 +512,37 @@ function _tasksAssignments_allow_only_add_timeline_action (phpQueryObject $doc)
   $actionDropdown->after($addTimelineBtn);
   $actionDropdown->remove();
 }
+
+/**
+ * Remove columns from timeline
+ *
+ * @param phpQueryObject $doc
+ */
+function _tasksAssignments_remove_columns_from_timeline ($doc) {
+  $selectColumnHeader = $doc->find('table th:nth-child(6)');
```

Comment:
Finding things like `table th:nth-child(6)` makes me a little nervous because the order might change, or a new column might be added and this will remove a column that is not intended.

Can we use `$doc->find('table th:contains("Status")` ? There are some benefits to this:

* What you are trying to select is more clear.
* It won't change the outcome if the order of the columns change or a new column is added. It also won't remove a random column if this happens.
* If the "Status" column is renamed, it will be visible again. That's the only drawback.

Reply:
I agree.